src/ipfs/ipfs-fetch.constant.ts

```diff
@@ -0,0 +1,80 @@
+import { IpfsGatewayTemplate } from '../types'
+
+export const DEFAULT_IPFS_GATEWAYS: IpfsGatewayTemplate[] = [
```

Comment:
i think we have too many gateways here. I'm not sure if this would affect the network performance when sent in parallel and results in a unideal result.

Reply:
i think we should select only popular and faster ones. Ideally ~5 gateways as default should be enough?

Reply:
Yes, you are right! Updated.